commit 0e80707d94e4c88f9879bdafcbaceb13432ec1f4
Author: Lorenzo Bianconi <lorenzo@kernel.org>
Date:   Tue Sep 6 16:36:32 2022 +0200

    net: ethernet: mtk_eth_soc: fix typo in __mtk_foe_entry_clear
    
    Set ib1 state to MTK_FOE_STATE_UNBIND in __mtk_foe_entry_clear routine.
    
    Fixes: 33fc42de33278 ("net: ethernet: mtk_eth_soc: support creating mac address based offload entries")
    Signed-off-by: Lorenzo Bianconi <lorenzo@kernel.org>
    Signed-off-by: David S. Miller <davem@davemloft.net>

--- a/drivers/net/ethernet/mediatek/mtk_ppe.c
+++ b/drivers/net/ethernet/mediatek/mtk_ppe.c
@@ -412,7 +412,7 @@ __mtk_foe_entry_clear(struct mtk_ppe *pp
 	if (entry->hash != 0xffff) {
 		ppe->foe_table[entry->hash].ib1 &= ~MTK_FOE_IB1_STATE;
 		ppe->foe_table[entry->hash].ib1 |= FIELD_PREP(MTK_FOE_IB1_STATE,
-							      MTK_FOE_STATE_BIND);
+							      MTK_FOE_STATE_UNBIND);
 		dma_wmb();
 	}
 	entry->hash = 0xffff;
